docs/how-to-guides/authentication-with-dex-gangway.md

```diff
@@ -191,7 +191,7 @@ HomePage URL must match the `issuer_host` in Dex configuration.
 
 Set **Authorization Callback URL** to the value of the `redirect_uri` in the Dex configuration.
 
-![Registering OAuth application ](github-oauth-app-register.png?raw=true "Register a new OAuth
+![Registering OAuth application ](../images/github-oauth-app-register.png?raw=true "Register a new OAuth
```

Comment:
Should this be part of this commit? It doesn't seem so.

Reply:
part of a different commit